devops/aws-codebuild/Jenkinsfile.cd

```diff
@@ -295,30 +295,6 @@ def macOSPipeline = {
                     sh './mac.01.libindy.setup.sh'
```

Comment:
if this is still needed (which I think it is) we should rename it (we no longer build libindy)